src/types/mod.rs

```diff
@@ -327,6 +370,18 @@ pub struct AccountInfo {
     pub account_stake:            Option<AccountStakingInfo>,
     /// Canonical address of the account.
     pub account_address:          AccountAddress,
+
+    /// The stake on the account that is in cooldown.
+    /// There can be multiple amounts in cooldown that expire at different
+    /// times.
+    pub cooldowns: Vec<Cooldown>,
```

Comment:
**Suggestion:** I think we need to document how this field behaves for blocks baked prior to P7.
I assume it will be empty.

Reply:
Yes, it will assume the default protobuf value if missing, i.e. an empty list.